pandas/core/generic.py

```diff
@@ -9680,6 +9681,19 @@ def last(self, offset) -> Self:
         2018-04-13  3
         2018-04-15  4
 
+        This is an alternative example. If you are not sure the index is sorted.
```

Comment:
Can you add this example at the very end of the docstring please? I think the comment below should stay and give context to the original `last` example.

I know we're adding this because of the deprecation, but I'd personally prefer to phrase this as `This is equivalent to...`